views/helpers.go

```diff
@@ -32,7 +31,6 @@ type (
 
 func (v *Views) getData(s *sessions.Session) *Context {
```

Comment:
Apologies, I'm still going to ask you to rename this function - `getData` isn't very descriptive (what data does it get and from where?)

Reply:
Renamed to getSessionData